packages/inter-protocol/src/price/fluxAggregatorKit.js

```diff
@@ -180,6 +180,9 @@ export const prepareFluxAggregatorKit = async (
         }),
       ),
   });
+  const now = await E(timerPresence).getCurrentTimestamp();
```

Comment:
I don't think we can put a remote call like that in the critical path to defining this kind. If this doesn't cause a null-upgrade test to fail, I think we're missing some coverage.

> Define all exo classes/kits before any incoming method calls from other vats -- in the first "crank".
 -- https://docs.agoric.com/guides/zoe/contract-upgrade.html#crank

How about a `helper.startInitializingVstorage(timerPresence)` thing that returns synchronously but asynchronously gets a timestamp and initializes vstorage?